src/image-copy/azext_imagecopy/__init__.py

```diff
@@ -46,6 +46,8 @@ def load_arguments(self, _):
             c.argument('timeout', options_list=['--timeout'], type=int, default=3600,
                        help='Time in seconds for the copy operation to finish. Increase this time if '
                        'you are going to copy large images (disks) like 512GB or more.')
+            c.argument('temporary_rg_name', options_list=['--temp-rg-name'], default='image-copy-rg',
```

Comment:
please rename the option and var name without abbreviations: **temporary_resource_group_name**